### ISSUE TITLE
Feed left margin

### ISSUE THREAD

**Issue by RasheedAZ** [degenrocket/degenrocket-web#3]
The left margin for the feed appears to be extremely narrow both on mobile and desktop. Tested on Firefox and Chromium with Rebelnet and Degenrocket spaces.

**Comment by degenrocket** [comment]
That's true, thanks. Added a small padding, let me know if it's still too narrow on your device. db91b6969d4fe7777e634739f955e45326866291
